PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+## Checklist
+- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
+- [x] No error nor warning in the console.
+## New functionality:
```

Comment:
Add some list items below here

Reply:
Something like:
## Description
This PR does blah-blah

## Items
- [ ] Fix smth;
- [ ] Add something blahblah;